frontend/src/features/Train/features/ObjectDetection/constants/detectionConstants.ts

```diff
@@ -0,0 +1,232 @@
+        display_name: "Random Horizontal Flip",
+        objectName: "transforms.RandomHorizontalFlip",
+        parameters: [
+          {
+            label: "probability",
+            index: 0,
+            parameter_name: "prob",
+            min: 0,
+            max: 1,
+            type: "number",
+          },
+        ],
+        label: "Random Horizontal Flip",
+        value: "RandomHorizontalFlip",
+      },
+    
+      ToTensor: {
+        display_name: "To Tensor",
+        objectName: "transforms.ToTensor",
+        parameters: [],
+        label: "To Tensor",
+        value: "ToTensor",
+      },
+
+      RandomVerticalFlip: {
+        display_name: "Random Vertical Flip",
+        objectName: "transforms.RandomVerticalFlip",
+        parameters: [
+          {
+            label: "probability",
+            index: 0,
+            parameter_name: "prob",
+            min: 0,
+            max: 1,
+            type: "number",
+          },
+        ],
+        label: "Random Vertical Flip",
+        value: "RandomVerticalFlip",
+      },
+      Resize: {
+        display_name: "Resize",
+        objectName: "transforms.Resize",
+        parameters: [
+          {
+            label: "Height",
+            min: 1,
+            max: 1000,
+            required: true,
+            type: "number",
+          },
+          {
+            label: "Width",
+            min: 1,
+            max: 1000,
+            required: true,
+            type: "number",
+          },
+        ],
+        label: "Resize",
+        value: "Resize",
+      },
+      GaussianBlur: {
+        display_name: "Gaussian Blur",
+        objectName: "transforms.GaussianBlur",
+        parameters: [
+          {
+            label: "kernel size",
+            index: 0,
+            parameter_name: "kernel size",
+            min: 1,
+            max: 1000,
+            type: "number",
+          },
+        ],
+        label: "Gaussian Blur",
+        value: "GaussianBlur",
+      },
+      Grayscale: {
+        display_name: "Grayscale",
+        objectName: "transforms.Grayscale",
+        parameters: [],
+        label: "Grayscale",
+        value: "Grayscale",
+      },
+      Normalize: {
+        display_name: "Normalize",
+        objectName: "transforms.Normalize",
+        parameters: [
+          {
+            label: "mean",
+            index: 0,
+            parameter_name: "mean",
+            min: -1000,
+            max: 1000,
+            default: 0,
+            type: "number",
+          },
+          {
+            label: "std",
+            index: 1,
+            parameter_name: "std",
+            min: -1000,
+            max: 1000,
+            default: 1,
+            type: "number",
+          },
+        ],
+        label: "Normalize",
+        value: "Normalize",
+      },
+      AdjustContrast: {
+        display_name: "AdjustContrast",
+        objectName: "transforms.functional.adjust_contrast",
+        transform_type: "functional",
```

Comment:
Use camelCase for object properties for consistency purposes. Also, what is transform_type used for?

Reply:
Pytorch has a category of transforms called functional transforms. In the old implementation, I used this parameter so that the backend could recognize what type of transform needed to be used.

Reply:
I removed it for now